gamestonk_terminal/economy/fred_view.py

```diff
@@ -1,194 +1,131 @@
 """ Fred View """
 __docformat__ = "numpy"
 
-import argparse
-from typing import List
-import pandas as pd
+import os
 from pandas.plotting import register_matplotlib_converters
+import pandas as pd
 import matplotlib.pyplot as plt
-from fredapi import Fred
-from gamestonk_terminal.helper_funcs import (
-    parse_known_args_and_warn,
-    valid_date,
-    plot_autoscale,
-)
+from gamestonk_terminal.economy import fred_model
+from gamestonk_terminal.helper_funcs import plot_autoscale, export_data
 from gamestonk_terminal.config_plot import PLOT_DPI
-from gamestonk_terminal import config_terminal as cfg
 from gamestonk_terminal import feature_flags as gtff
 
 register_matplotlib_converters()
 
 
-fred_series_description = {
-    "VIXCLS": "Volatility Index",
-    "GDP": "Gross Domestic Product",
-    "UNRATE": "Unemployment Rate",
-    "DGS1": "1-Year Treasury Constant Maturity Rate",
-    "DGS5": "5-Year Treasury Constant Maturity Rate",
-    "DGS10": "10-Year Treasury Constant Maturity Rate",
-    "DGS30": "30-Year Treasury Constant Maturity Rate",
-    "MORTGAGE30US": "30-Year Mortgage Rate",
-    "FEDFUNDS": "Effective Federal Funds Rate",
-    "AAA": "Moody's Seasoned AAA Corporate Bond Yield",
-    "DEXCAUS": "Canada / U.S. Foreign Exchange Rate",
-}
+def notes(series_term: str, num: int):
```

Comment:
Export option?

Reply:
This is not a dataframe, hence why didn't do it. Should we convert to one and do it?

Reply:
I'm just keeping with your export all commands theme lol.  I don't see the need for an export

Reply:
All commands that have dataframe with data as output. Commands without dataframe/series I don't think should be exportable